main.cpp

```diff
@@ -19,13 +19,24 @@ using namespace clang::tooling;
 
 class CastCallBack : public MatchFinder::MatchCallback {
 public:
-    CastCallBack(Rewriter& rewriter) {
+    CastCallBack(Rewriter& rewriter) : _rewriter(rewriter) {
         // Your code goes here
     };
 
     void run(const MatchFinder::MatchResult &Result) override {

```

Comment:
This code does not work as expected for the following example:
```cpp
// test.cpp
int main() {
    float fooooo;
    int i = (int)   fooooo   ;
    return 0;
}
```

Reply:
Made a commit that fixed it
![2022-05-27_17-42-57](https://user-images.githubusercontent.com/61184952/170723088-14d78f7b-8815-41d4-babe-cf29d95ff9ab.png)